docs/plugin-system.md

```diff
@@ -36,13 +36,13 @@ A basic plugin could be something like
     <script src="MetaflowPluginAPI.js"></script>
     <script>
       (function () {
-        Metaflow.register('run-header', () => {
+        Metaflow.onReady((configuration, resource) => {
```

Comment:
below it mentions `There are two implemented plugin slots, `run-header` and `task-details`. The desired slot must be given as parameter to a plugin API register message.` but here it removes run-header registry call - is there some equivalent of it?

Reply:
Updated to have mention about manifest.json